src/polyfills.ts

```diff
@@ -0,0 +1,5 @@
+// create an alias for the crypto api for IE11
```

Comment:
Maybe also mention the library that needs this, so we know that we can remove this in case we don't use `nanoid` anymore.

Reply:
that makes sense, I've added it to the comment